ddp-workspace/projects/toolkit/src/lib/components/age-up/verifyAgeUpPage.component.ts

```diff
@@ -17,20 +17,20 @@ import { ToolkitConfigurationService } from '../../services/toolkitConfiguration
     </main>`
 })
 export class VerifyAgeUpPageComponent implements OnInit {
-    constructor(
-        private router: Router,
-        private headerConfig: HeaderConfigurationService,
-        @Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService) { }
+  constructor(
+    private router: Router,
+    private headerConfig: HeaderConfigurationService,
+    @Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService) { }
 
-    public ngOnInit(): void {
-        this.headerConfig.setupDefaultHeader();
-    }
+  public ngOnInit(): void {
+    this.headerConfig.setupDefaultHeader();
+  }
 
-    public handleNextUrl(url: string): void {
-        this.router.navigateByUrl(url);
-    }
+  public handleNextUrl(url: string): void {
+    this.router.navigateByUrl(url);
+  }
 
-    public handleError(): void {
-        this.router.navigateByUrl(this.toolkitConfiguration.errorUrl);
-    }
+  public handleError(): void {
+    this.router.navigateByUrl(this.toolkitConfiguration.errorUrl);
+  }
```

Comment:
These file disappeared when I merged origin/develop into this branch. I have no idea why that is, so I am adding it again brute force.